addons/advanced_fatigue/functions/fnc_handleStaminaBar.sqf

```diff
@@ -31,6 +31,8 @@ if (GVAR(fadeStaminaBar)) then {
     } else {
         _staminaBarContainer ctrlSetFade (0.9 * _stamina / 0.8);
     };
+} else {
+    _staminaBarContainer ctrlSetFade 0;
```

Comment:
Why?

Reply:
With `fadeStaminaBar` set to false, ctrlFade isn't restored to 0 after death or initial respawn screen.

Reply:
Hmm I guess two more commands doesn't really hurt.